### PR TITLE
procedures: List, create, start, stop, and remove one or more workspaces by using APIs

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Add new sections that explain Che admins how to list, create, start, stop, and remove one or more workspaces by using APIs.

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-2160
https://issues.redhat.com/browse/RHDEVDOCS-3216
https://issues.redhat.com/browse/RHDEVDOCS-4093

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Comment by max-cx** [comment]
Closing this PR because it is continued as https://github.com/eclipse-che/che-docs/pull/2510.

**Comment by max-cx** [comment]
Reopening for Nick to test his GitHub settings.

File: modules/end-user-guide/pages/managing-workspaces-with-apis.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+:description: Managing workspaces with OpenShift APIs
+:keywords: api, list workspaces, create workspace, restart workspace, stop workspace, start workspace, remove workspace
+// :page-aliases:
+
+[id="managing-workspaces-with-apis"]
+= Managing workspaces with OpenShift APIs
+
+...
+
+Consider writing this entire page as a reference. (If so, then rename all the files in `/partials/` and in the `include::` statements below from `proc_` to `ref_`.)
```
Comment (nickboldt) [inline modules/end-user-guide/pages/managing-workspaces-with-apis.adoc:12]:
```suggestion
Consider writing this whole page as a reference. (If so, then rename all the files in `/partials/` and in the `include::` statements below from `proc_` to `ref_`.)
```

File: modules/end-user-guide/pages/managing-workspaces-with-apis.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+:description: Managing workspaces with OpenShift APIs
+:keywords: api, list workspaces, create workspace, restart workspace, stop workspace, start workspace, remove workspace
+// :page-aliases:
+
+[id="managing-workspaces-with-apis"]
+= Managing workspaces with OpenShift APIs
+
+...
+
+Consider writing this whole page as a reference. (If so, then rename all the files in `/partials/` and in the `include::` statements below from `proc_` to `ref_`.)
```
Comment (nickboldt) [inline modules/end-user-guide/pages/managing-workspaces-with-apis.adoc]:
```suggestion
Consider writing this entire page as a reference. (If so, then rename all the files in `/partials/` and in the `include::` statements below from `proc_` to `ref_`.)
```

File: modules/end-user-guide/pages/managing-workspaces-with-apis.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+:navtitle: Managing workspaces with OpenShift APIs
+:description: Managing workspaces with OpenShift APIs
+:keywords: api, list workspaces, create workspace, restart workspace, stop workspace, start workspace, remove workspace
+// :page-aliases:
+
+[id="managing-workspaces-with-apis"]
+= Managing workspaces with OpenShift APIs
```
Comment (max-cx) [inline modules/end-user-guide/pages/managing-workspaces-with-apis.adoc:8]: @dkwon17, just to clarify: `OpenShift APIs` or `{orch-name} APIs`?

**Comment by max-cx** [comment]
Closing. Hopefully it was useful to Nick for testing his GitHub settings.

**Pull request closed without merging**